.ddev/.disabled-services/docker-compose.chromedriver.yaml

```diff
@@ -0,0 +1,30 @@
+---
+# Docker-ChromeDriver configuration for DDEV-Local.
+# There is one item that must be configured in this file in order for it to
+# work, please see below.
+
+version: '3.6'
+services:
+  chromedriver:
+    image:  previousnext/chrome-headless:65
```

Comment:
Is this really referring to Chrome 65? I don't see a repo for https://hub.docker.com/r/previousnext/chrome-headless/tags.

... looks like it:

```console
docker run --rm -it previousnext/chrome-headless:65 /opt/google/google-chrome --version
Google Chrome 65.0.3325.162
```

Reply:
I'm happy to get off of that image. I. tried once and couldn't get to green.

Reply:
I filed https://massgov.atlassian.net/browse/DP-24306 for this given Chrome is up to version 99.